template/README.md

```diff
@@ -27,7 +27,7 @@ Make sure that **docker** is installed.
 
 Clone the project from the analytics Models repo.
 ```
-git clone git@github.com:NeowayLabs/data-science-template.git
+git clone https://github.com/<@github_username>/<@project>.git
```

Comment:
Do you think it's more easier for beginners init with ssh or https protocol?

Reply:
Https for sure. If i'm not mistaken, you have ve to have ssh setup to clone with git@...

Reply:
Yes.